openpype/hosts/fusion/plugins/publish/validate_saver_output_extension.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin,
+)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.fusion.api.action import SelectInvalidAction
+
+
+class ValidateSaverOutputExtension(
+    pyblish.api.InstancePlugin, OptionalPyblishPluginMixin
+):
+    """
+    Validate Saver Output Extension matches Publish menu
+
+    This ensures that if the user tweaks the 'Output File Extension' in the publish menu,
+    it is respected during the publish.
```

Comment:
Honestly - this what is explained here - doesn't that make more sense to be behavior on the Creator?

Reply:
By 'Creator' do you mean the Create menu? If so you don't have access to the attributes of the 'Render' variant in that menu (unless I'm missing something), you instead just have access to the new variants settings.

Reply:
Ah actually I think I might know what you mean, you can select the Variant and tweak there. On our internal build I think this code has been tweaked to use this field for something else, I'll create a fresh build to test this approach.

Reply:
I'm not saying the variant - but the creator attributes below it.
If you want to allow updating the image format on the Creator after creation you'd implement `get_attribute_defs` on the creator instead of only `get_pre_create_attr_defs`. Does that help with anything?

Reply:
I've just made some new changes that tweak the update conditions which essentially removes the need for a validator here. Let me know what you think of the new implementation.